### PR TITLE
OWF migration prep

### LINKED ISSUE #684: OWF Migration
Go through the project and update document, naming and code references for the migration of the project to OWF.

### PR DESCRIPTION
**Pull request by esune**
This PR includes changes related to the future migration of the project to OWF:
* Removes custom GHA responsible for pushing deployments to BCGov-owned environments
* Renames the project to `acapy-vc-authn-oidc`
* Renames components to drop the `aries` naming where possible
* Provides a new OWF-like skin for webpages

Note:
* The chart name and variables were left as `vc-authn` as it would cause a lot of overhead to replace everything, for little to no value (in my opinion)
* Existing documentation retains the VC-AuthN nomenclature if it does not cause issues with understanding the content

~Still in the works:~
* ~QR Code page re-skinning~
* ~Proof-request configuration explorer re-skinning~

Resolves #684

### CODE REVIEW

**Comment by esune** [comment]
A couple more things that will require attention, either in this PR or shortly after:
* We need to be able to dynamically mount a custom QR code page and related assets. Currently the Helm chart does not support this, and the code only supports using custom HTML (not adding/replacing assets).
* We need to store a backup of the existing pages to be reused for BCGov deployments.
* Wording in the QR code page revolves around the use of BC Wallet: how can we update this to be less specific?

**Comment by loneil** [comment]
Additional consideration for this repo maybe along with the 3 above. The Readme has curl commands to set up proof specific to the BC Wallet person cred check.

**Review by loneil — COMMENTED** [review]
I tested this out locally against an existing build and with a from-scratch one, things still working as expected for the app and demo.

If we're taking out the push-to-main action build would we need releases to go to our `dev` env or can we still have dev running of a `main` build?

Not too important since it's just the demo but the logo colors look like:
image

**Comment by esune** [comment]
> I tested this out locally against an existing build and with a from-scratch one, things still working as expected for the app and demo.
> 
> If we're taking out the push-to-main action build would we need releases to go to our `dev` env or can we still have dev running of a `main` build?
> 
> Not too important since it's just the demo but the logo colors look like: image

For the update strategy for our `dev`: I am not sure, we will need to figure out what the best strategy is. We could set-up either way, just not from the repo directly I think.

For the UI I will try and tweak colors, I just tried to make it "acceptable" as a first pass 😄

**Comment by esune** [comment]
I pushed a couple more tweaks, including bumping the chart version to a different minor to more clearly indicate the shift.
UI tweaks were basic and targeting a general rebranding, more tweaks/work can be done once the project has moved.

File: demo/vue/app/config/test.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "logLevel": "silent"
   },
   "serviceClient": {
-    "commonServices": {
+    "customService": {
       "username": "username",
       "password": "password"
     }
   }
```
Comment (loneil) [inline demo/vue/app/config/test.json:21]: This whole block can go (serviceClient can be unused)

**Review by loneil — COMMENTED** [review]
Minor (and in the demo) but the files `chesService` and `cdogsService` can be removed as they reference BCGov common services from the scaffold app.
Then any refernce to `serviceClient` config is unneeded too.

File: README.md [hunk 2]
```diff
@@ -58,24 +57,19 @@ curl -X 'POST' \
...
+        "names": ["attr1", "attr2", "attr3"],
         "restrictions": [
           {
-              "schema_name": "Person",
-              "issuer_did": "QEquAHkM35w4XVT3Ku5yat"
-          },
-          {
-              "schema_name": "Person",
-              "issuer_did": "RGjWbW1eycP7FrMf4QJvX8"
+              "schema_name": "test-schema",
```
Comment (loneil) [inline README.md:72]: Get an error running with this trailing comma

`{"detail":[{"type":"json_invalid","loc":["body",359],"msg":"JSON decode error","input":{},"ctx":{"error":"Expecting property name enclosed in double quotes"}}]}`

**Review by loneil — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (esune) [reply README.md]: It should be fixed now

**Review by loneil — APPROVED** [review]

**Comment by esune** [comment]
Thanks for the review @loneil!

@WadeBarnes I think we could plan the next steps of the move now.

**Comment by WadeBarnes** [comment]
Coordinating the move.

**Comment by WadeBarnes** [comment]
I'll merge this PR right before the move.

**Pull request merged**